docs/src/app/components/pages/customization/themes.md

```diff
@@ -0,0 +1,232 @@
+
+Get theme whenever you need to use it in your own components:
+
+```js
+import React from 'react';
+
+class DeepDownTheTree extends React.Component {
+
+  static contextTypes = {
+    muiTheme: React.PropTypes.object,
+  };
+
+  render () {
+    return (
+      <span style={{color: this.context.muiTheme.baseTheme.palette.textColor}}>
+        Hello World!
+      </span>
+    );
+  }
+}
+
+export default DeepDownTheTree;
+```
+
+## API
+
+The items listed below are everything related to how Material-UI's theme work.
+
+### `getMuiTheme(baseTheme, themeOverrides) => muiTheme`
+
+This function takes in the `baseTheme` merges in onto the default base theme that is the 
+[lightBaseTheme](https://github.com/callemall/material-ui/blob/master/src/styles/baseThemes/lightBaseTheme.js)
+and calculates the muiTheme from it. In other words anything you don't specify on the
+`baseTheme` object will be picked up from the `lightBaseTheme`.
+
+Keep in mind, any changes to the theme object must appear as another call to this function.
+**Never** directly mutate the theme as the effects will not be reflected in any component
+until another render is triggered for that component leaving your application in a moody state.
+
+The `baseTheme` object looks like this (these are the defaults):
+
+```js
+import Colors from 'material-ui/lib/styles/colors';
+import ColorManipulator from 'material-ui/lib/utils/color-manipulator';
+
+const baseTheme = {
+  spacing: {
+    iconSize: 24,
+    desktopGutter: 24,
+    desktopGutterMore: 32,
+    desktopGutterLess: 16,
+    desktopGutterMini: 8,
+    desktopKeylineIncrement: 64,
+    desktopDropDownMenuItemHeight: 32,
+    desktopDropDownMenuFontSize: 15,
+    desktopLeftNavMenuItemHeight: 48,
+    desktopSubheaderHeight: 48,
+    desktopToolbarHeight: 56,
+  },
+  fontFamily: 'Roboto, sans-serif',
+  palette: {
+    primary1Color: Colors.cyan500,
+    primary2Color: Colors.cyan700,
+    primary3Color: Colors.grey400,
+    accent1Color: Colors.pinkA200,
+    accent2Color: Colors.grey100,
+    accent3Color: Colors.grey500,
+    textColor: Colors.darkBlack,
+    alternateTextColor: Colors.white,
+    canvasColor: Colors.white,
+    borderColor: Colors.grey300,
+    disabledColor: ColorManipulator.fade(Colors.darkBlack, 0.3),
+    pickerHeaderColor: Colors.cyan500,
+    clockCircleColor: ColorManipulator.fade(Colors.darkBlack, 0.07),
+    shadowColor: Colors.fullBlack,
+  },
+};
+```
+
+The second argument can be used to override the values calculated from the `baseTheme`.
+To see what are the values you can override with this argument. Use the
+[source](https://github.com/callemall/material-ui/blob/master/src/styles/getMuiTheme.js#L23-L262)
+, Luke...
+
+### `<MuiThemeProvider/>`
+
+This component takes a theme as a property and passes it down with context.
+This should preferably be at the root of your component tree. The first
+example demonstrates it's usage.
+
+### `muiThemable() => ThemeWrapper(Component) => WrappedComponent`
```

Comment:
Not to call @mbrookes out (sorry!) Since we got a good first draft of this, maybe we can have him to a quick run through after we merge for any other grammar/spelling/word improvements he finds.